src/soot/jimple/spark/solver/PropWorklist.java

```diff
@@ -69,6 +69,9 @@ public final void propagate() {
 						public final void visit(Node n) {
 							AllocDotField nDotF = pag.makeAllocDotField(
 									(AllocNode) n, target.getField());
+                            if(ofcg != null) {
```

Comment:
I have not been able to figure out the indentation style preferred by Ondrej in the Spark code, so the indentation in this change is inconsistent. Let me know the appropriate scheme to use and I will happily adjust.